File: DistributionWorker/TasksBase/MessageTypes.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TasksBase
{
    public static class MessageTypes
    {
        public const string START = "START";
        public const string STOP = "STOP";
        public const string ENDED_TASK = "ENDED_TASK";
        public const string RECONNECT = "RECONNECT";
    }
}

File: DServerBase/src/main/java/tasks/MessageTypes.java
package tasks;

public interface MessageTypes {
    // Required fields - ended - boolean, answer - if ended, answerCount - if answered, answer{i} - i answer
    public static String ENDED_TASK = "ENDED_TASK";
    public static String RECONNECT = "RECONNECT";
    // Required fields - taskId
    public static String START = "START";
    public static String STOP = "STOP";
}
